parrots.json

```diff
@@ -643,5 +643,9 @@
     {
       "hd": "hd/backwardsparrot.gif",
       "name": "Backwards Parrot"
+    },
+    {
+      "gif":"ripparrot.gif",
+      "name":"RIP Parrot",
```

Comment:
Looks like the comma on this line is causing tests and deployment to fail.

Reply:
removed!